src/lib/holocene/button.stories.svelte

```diff
@@ -73,22 +73,6 @@
         },
         if: { arg: 'href' },
       },
-      borderModifier: {
-        name: 'Border Modifier',
-        control: 'select',
-        options: ['borderless', 'borderless-left', 'borderless-right'],
-        table: {
-          category: 'Modifiers',
-        },
-      },
-      borderRadiusModifier: {
-        name: 'Border Radius Modifier',
-        control: 'select',
-        options: ['square', 'square-left', 'square-right'],
-        table: {
-          category: 'Modifiers',
-        },
-      },
```

Comment:
There's a few places we're using this where the button is next to an input, will need to account for that

Reply:
This is only removing the control from Storybook, not the actual functionality. I figure the stories where it's used should capture the functionality. A random knob that lets you make the left-side of a button square isn't _super useful_. But, I can be persuaded to put it back.